calendar-app/App.js

```diff
@@ -3,34 +3,56 @@ import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { View, StyleSheet } from 'react-native';
 import { GlobalColor, GlobalSecondaryColor, GlobalFont } from './Styles';
 import RegistrationScreen from './Screens/RegistrationScreen';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import Footer from './Components/Footer';
-import { getIsLoggedIn } from './Components/Storage/userDataStorage'; 
-import { useEffect } from 'react';
+import { getIsLoggedIn, getUserId, storeUserInfo } from './Components/Storage/userDataStorage'; 
 import { Provider } from 'react-native-paper';
+import { getUserById } from './Components/API/Users/UserGetById';
 
+async function getUser() {
+  const userId = await getUserId();
+  return userId;
+}
 
 const App = () => {
   const [refreshKey, setRefreshKey] = useState(0);
   const [isRegistered, setIsRegistered] = useState(false);
+  const [userId, setUserId] = useState(null);
   
   const handleRefresh = () => {
     setRefreshKey(oldKey => oldKey + 1);
     setIsRegistered(true);
   };
 
   useEffect(() => {
-    const checkLoginStatus = async () => {
-      const loggedIn = await getIsLoggedIn();
-      if (loggedIn) {
-        setIsRegistered(true);
-      }
-    };
-
+    getUser().then(setUserId);
     checkLoginStatus();
   }, []);
 
+  useEffect(() => {
+    if (userId) { 
+      updateUserData();
+    }
+  }, [userId]); 
+  
+  const updateUserData = async () => {
+    const userData = await getUserById({ userId });
+    if(userData != null) {
+      if(userData.group_id !== null) {
```

Comment:
Just keep in mind, consistency is best shi. 40 line has != 41 line has !==

Reply:
Updated it in the latest commit